docs/DeveloperGuide.md

```diff
@@ -80,6 +81,17 @@ We use [Cypress](https://www.cypress.io/) for automated end-to-end front-end tes
 1.  At project root, run `gradlew frontendTest`
 > If you encountered an invalid browser error, please ensure that you have `Chrome` installed in the default installation directory. Otherwise, follow the instructions [here](https://docs.cypress.io/guides/guides/debugging.html#Launching-browsers) to create symbolic links so Cypress can locate `Chrome` in your system.
 
+### (Optional) Using Vue.js devtools for frontend debugging
+1. View and download [Vue.js devtools](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd) extention on Chrome web store.
```

Comment:
exten~t~`s`ion 
~View and download~ Using Chrome, visit the website ... and add the extension to your Chrome.